src/js/components/AccordionPanel/AccordionPanel.js

```diff
@@ -10,11 +10,19 @@ import { AccordionContext } from '../Accordion/AccordionContext';
 import { AccordionPanelPropTypes } from './propTypes';
 import { useThemeValue } from '../../utils/useThemeValue';
 
+// Ideally we should use `useId` from react but this is only available
+// in React 18 and we want to keep compatibility with React 17 and 16.
+function getUID() {
+  const timestamp = Date.now().toString(36);
```

Comment:
Should we do:

`Date.now() + Math.random()` instead of just appending Math.random?

https://stackoverflow.com/questions/8012002/create-a-unique-number-with-javascript-time/40591207#40591207

Reply:
Changed in latest commit